doc/developer-guide.rst

```diff
@@ -0,0 +1,45 @@
+.. py:currentmodule:: lsst.ts.MTDomeTrajectory
+
+.. _lsst.ts.MTDomeTrajectory.developer_guide:
+
+###############
+Developer Guide
+###############
+
+The MTDomeTrajectory CSC is implemented using `ts_salobj <https://github.com/lsst-ts/ts_salobj>`_.
+
+.. _lsst.ts.MTDomeTrajectory-api:
+
+API
+===
+
+The primary classes are:
+
+* `MTDomeTrajectory`: the CSC.
+* `BaseAlgorithm`: base class for following algorithms.
+* `SimpleAlgorithm`: a simple following algorithm.
+
+.. automodapi:: lsst.ts.MTDomeTrajectory
+   :no-main-docstr:
+
+.. _lsst.ts.MTDomeTrajectory-build_and_test:
+
+Build and Test
+==============
+
+This is a pure python package. There is nothing to build except the documentation.
+
+.. code-block:: bash
+
+    make_idl_files.py MTDomeTrajectory
+    setup -r .
+    pytest -v  # to run tests
+    package-docs clean; package-docs build  # to build the documentation
+
+.. _lsst.ts.MTDomeTrajectory-contributing:
+
+Contributing
```

Comment:
On the top-level page, and also probably here, it'd be good to link where the people involved with the CSC can be found.
https://ts-xml.lsst.io/index.html#index-master-csc-table-mtdometrajectory

I see that you're listed in the XML as both the PO and main developer. That should get updated (not necessarily in this ticket). I think Sandrine is the PO...

Reply:
I would rather not duplicate that information here. It is certainly going to go out of date.

Reply:
I wonder if we have our wires crossed. How is adding a link to the table duplicating information?

Reply:
Got it. I added an icon labeled "Project Metadata" to the main page.